### DISCUSSION TITLE
Horizontal Scrolling on Rating section: SC 1.4.10 Reflow

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3233] [Q&A] [answered]
Hi Everyone,

I have a scenario where after applying the reflow settings, horizontal scrolling appears on the page.

Can we consider it as a failure or it is not a failure of 1.4.10 since it can be necessary to keep a horizontal scroll bar to view the rating content?

This is a screenshot of a 100 % view
![image](https://github.com/w3c/wcag/assets/115781693/3f0cbbcb-d76e-4704-a227-d8814308f77c)

This is a screenshot after applying the reflow settings.

![image](https://github.com/w3c/wcag/assets/115781693/757f3a5b-39dd-4fce-8696-20953204d649)

**Comment by JAWS-test** [comment]
Related: https://github.com/w3c/wcag/issues/2073

**Comment by JAWS-test** [comment]
I think the horizontal scrolling of the area is ok because it does not need to be scrolled vertically at the same time

**Reply by AccessibilityEnthusiastic** [reply]
I have a new scenario where I need to scroll vertically and horizontally to view the content. Will it be a failure of Reflow?
This is a screenshot of a 100 % view:
![image](https://github.com/w3c/wcag/assets/115781693/6b64337f-20cb-4629-8411-1e03ab15d056)

This is a screenshot after applying the reflow settings:
![image](https://github.com/w3c/wcag/assets/115781693/af63a88a-c2d2-41cd-9d3e-107dba66dc76)

**Reply by JAWS-test** [reply]
It's impossible to decide from your screenshot, because there are scrollbars, but no content that needs to be scrolled horizontally - so that would be ok. But I suspect that further down there is content that doesn't reflow and needs to be scrolled horizontally (probably the links) and that would then be a violation of the SC

**Reply by AccessibilityEnthusiastic** [reply]
Why we are not considering the rating section as a violation?
There also we need to scroll horizontally to view the rating content.

**Comment by JAWS-test** [comment]
> Why we are not considering the rating section as a violation? There also we need to scroll horizontally to view the rating content.

More people should comment on this. My understanding of 1.4.10 would be that two-dimensional scrolling is prohibited. However, since the rating section only scrolls in one direction, that is fine.

**Comment by patrickhlauke** [comment, marked as answer]
I believe recent changes/additions to 1.4.10 Reflow understanding cover these sorts of scenarios a bit more clearly now
